projects/angular-sdk/src/components/global.component.ts

```diff
@@ -58,6 +61,10 @@ export class GlobalComponent implements OnChanges {
     // Life Cycle Hooks
     // -----------------------------------------------
 
+    ngOnInit(): void {
+        this.logger.setEnv(this.env)
```

Comment:
Don't forget the ';' ;) (until we have a automatic check for that)

Reply:
ah keep missing it. updated